### DISCUSSION TITLE
最新版本是否还能通过正向ws连接 go-cqhttp？

### DISCUSSION THREAD

**Discussion by RinChanNOWWW** [nonebot/discussions#362] [问答] [answered]
曾经有一个 CQHTTP_WS_URLS 配置，现在貌似没有了？文档中也找不到。

**Comment by RinChanNOWWW** [comment]
通过 https://github.com/orgs/nonebot/discussions/86 发现了新的配置 key。

**Comment by yanyongyu** [comment, marked as answer]
https://onebot.adapters.nonebot.dev

**Reply by RinChanNOWWW** [reply]
话说为什么更推荐反向连接呢

**Reply by yanyongyu** [reply]
nb作为服务端可以实现更多功能
